rest_framework_tracking/mixins.py

```diff
@@ -120,17 +120,25 @@ def _clean_data(self, data):
         You can define your own sensitive fields in your view by defining a set
         eg: sensitive_fields = {'field1', 'field2'}
         """
-        data = dict(data)
+        # data = dict(data)
 
         SENSITIVE_FIELDS = {'api', 'token', 'key', 'secret', 'password', 'signature'}
         CLEANED_SUBSTITUTE = '********************'
 
         if self.sensitive_fields:
             SENSITIVE_FIELDS = SENSITIVE_FIELDS | {field.lower() for field in self.sensitive_fields}
 
-        for key in data:
-            if key.lower() in SENSITIVE_FIELDS:
-                data[key] = CLEANED_SUBSTITUTE
+        if type(data) is list:
+            data = list(data)
+            for data_list in data:
+                for key in data_list:
+                    if key.lower() in SENSITIVE_FIELDS:
+                        data[key] = CLEANED_SUBSTITUTE
+        else:
+            data = dict(data)
+            for key in data:
+                if key.lower() in SENSITIVE_FIELDS:
+                    data[key] = CLEANED_SUBSTITUTE
```

Comment:
We duplicate a lot of code here, instead of doing what you do we could have:
`
if type(data) is not list:
    data_list = [data]
for data in data_list:
    d = dict(data)
    for key in d:
        if key.lower() in SENSITIVE_FIELDS:
            d[key] = CLEANED_SUBSTITUTE
return d`